### ISSUE TITLE
windowlist@cobinja.de: Previews show icon & text only, no image

### ISSUE THREAD

**Issue by vertigo220** [linuxmint/cinnamon-spices-applets#1653]
```
 * Applet version/Build date (to be found in "About.." or in "metadata.json") - 2018/01/08
 * Cinnamon version (cinnamon --version) - 3.6.7
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) - Mint 18.3
 * Graphics hardware *and* driver used - i3 integrated / unknown
 * Snippet or attachment of ~/.cinnamon/glass.log

[glass.log](https://github.com/linuxmint/cinnamon-spices-applets/files/1786485/glass.log)

 * Snippet or attachment of ~/.xsession-errors

[xsession-errors.txt](https://github.com/linuxmint/cinnamon-spices-applets/files/1786489/xsession-errors.txt)


 ```

**Notify author of applet**
@Cobinja 

**Issue**

Hovering over a button to view its preview only shows the icon and text as it's shown in the button, not a preview of the window.

**Steps to reproduce**

Hover over a button

**Expected behaviour**

The popup should show a preview of the window

**Other information**

**Comment by Cobinja** [comment]
Is your cinnamon running in Software Rendering mode?
If yes, it's by design to do that to minimize continuous rendering.

**Comment by vertigo220** [comment]
I'm not sure. I was accessing it through Chrome Remote Desktop, which I set to run in normal, not software rendering, mode, but there's no telling.

**Comment by Cobinja** [comment]
Does $HOME/.cinnamon/glass.log have an entry "Cinnamon Software Rendering mode enabled"?

**Comment by vertigo220** [comment]
Yeah. Not sure why it is when it shouldn't be, but it is. Maybe that should be an option in CWL, since other window list managers display the thumbnail and it doesn't seem to cause any issues. In any case, thanks for clearing it up.

I should probably make a new issue for this, but I also have a problem where the buttons keep aligning to the right side of the panel. Not sure what causes this or what fixes it, though it was just doing it and turning panel edit mode on, messing with it a bit, then turning it off caused them to align back to the left. I'm pretty sure this issue occurs with Icing Task Manager as well. And a feature request is to be able to select which apps to show labels/captions for and which to show just the icons. Should I make separate issues for these?

**Comment by Cobinja** [comment]
Please open that on my repository https://github.com/Cobinja/CobiWindowList

**Comment by Cobinja** [comment]
Tracked here: https://github.com/Cobinja/CobiWindowList/issues/14
